src/algebra/big_operators/order.lean

```diff
@@ -317,6 +317,19 @@ end canonically_ordered_comm_semiring
 
 end finset
 
+namespace fintype
+
+variables [fintype α]
+
+@[mono] lemma sum_mono [ordered_add_comm_monoid β] : monotone (λ f : α → β, ∑ x, f x) :=
+λ f g hfg, finset.sum_le_sum $ λ x _, hfg x
```

Comment:
This is in the `fintype` namespace since we already have lemmas called `fintype.sum_bool` instead of `finset.sum_univ_bool`